packages/eslint-plugin/tests/rules/ban-ts-comment.test.ts

```diff
@@ -1127,6 +1119,22 @@ if (false) {
         },
       ],
     },
+    {
+      // comment's column > first statement's column
+      // eslint-disable-next-line @typescript-eslint/internal/plugin-test-formatting
```

Comment:
I used an `eslint-disable` comment over `noFormat()` [since it doesn't handle disabling indentation checks](https://github.com/typescript-eslint/typescript-eslint/blob/94c5484f747be901f1610fc37192714a619bb355/packages/eslint-plugin-internal/src/rules/plugin-test-formatting.ts#L392-L398).